### ISSUE TITLE
Swift Package manager support

### ISSUE THREAD

**Issue by acarrera94** [RevenueCat/purchases-ios#125]
Xcode 11 comes with swift package manager, a lot easier than dealing with cocoapods or carthage, all of our other packages are on SPM, would love to see this one be there as well!

**Comment by acarrera94** [comment]
Here's a working `Package.swift` to get started with, just add the test frameworks 
```
// swift-tools-version:5.0
// The swift-tools-version declares the minimum version of Swift required to build this package.
import PackageDescription

let package = Package(
    name: "Purchases",
    products: [
        .library(name: "Purchases", targets: ["Purchases"])
    ],
    dependencies: [
        //    .package(url: "https://github.com/AliSoftware/OHHTTPStubs", .exact("8.0.0")),
        //    .package(url: "https://github.com/Quick/Nimble", .exact("8.0.4"))
    ],
    targets:[
    .target(name: "Purchases", dependencies: [], path: "Purchases", exclude: [], sources: nil, publicHeadersPath: nil)
//        .testTarget(name: "PurchasesTests", dependencies: []"Purchases"])
    ]
)

```

**Comment by acarrera94** [comment]
Looks like OHHTTPStubs is working on adding SPM as well.

**Comment by jeiting** [comment]
Hey Andre!

I'm not too familiar with SPM yet, but as long as it supports Objective-C frameworks we'd like to support it.

I'll leave this ticket open to track progress.

**Comment by vegaro** [comment]
Hi! I have pushed a branch `feature/spm-support` to add support for this. I have tested it and it looks fine, but it good be good if it could be tested by someone else. Can you take a look and let us know if it's working as expected?

Thanks!!

**Comment by acarrera94** [comment]
I’ll take a look this week!

**Comment by acarrera94** [comment]
@vegaro it looks good! Everything is working in my project.

**Comment by acarrera94** [comment]
It looks like the test library is breaking SwiftUI canvas. See https://github.com/sunshinejr/SwiftyUserDefaults/issues/211

**Comment by vegaro** [comment]
Interesting. Thanks for following up. It looks like we could do something like this https://github.com/Moya/Moya/pull/1923/files in the meantime... I will take a look this week

**Comment by cliftonlabrum** [comment]
Just adding my voice that official SPM support would be fantastic. 🙂I'll try the `feature/spm-support` branch mentioned above. Thanks!

**Comment by vegaro** [comment]
Just pushed an update to `feature/spm-support` and it looks like that SwiftUI canvas issue. Can you verify it works for you now @acarrera94 ?

**Comment by acarrera94** [comment]
Sure I’ll take a look tomorrow.

**Comment by acarrera94** [comment]
@vegaro Feature branch is working great including SwifUI canvas!

**Comment by vegaro** [comment]
I just released 2.6.1 that should include SPM support. Thanks for collaborating on this issue.

**Comment by cliftonlabrum** [comment]
@vegaro Thank you for this! It might be worth mentioning in the **Installation** section of the readme that SPM is an option. 🙂
